wpeverest/user-registration
# UR-481 Fix - Errors and warnings in the console

### All Submissions:






### Changes proposed in this Pull Request:



Previously, when fields are used multiple times then it shows non-unique id warnings and jQuery image not found warnings in console. This PR will these issues.

### How to test the changes in this Pull Request:

1. Drag and Drop fields multiple times and update the form.
2. then reload the form and open console.
3. then verify whether the issues have been fixed or not.

### Types of changes:




### Other information:

* [  ] Have you updated the documentation accordingly?



### Changelog entry

> Fix - Errors and warnings in the console.
